### ISSUE TITLE
enforceMorphMap

### ISSUE THREAD

**Issue by lee-to** [laravel/framework#46730]
### Laravel Version

10

### PHP Version

8.1

### Description

In the documentation in the chapter on custom polymorphic types, the enforceMorphMap method is indicated, which causes an error if you do not also add an alias for the User model but call auth()->user()->unreadNotifications 

Error message:
No morph map defined for model [App\Models\User].

But it gets fixed if you use the method morphMap 

But this is not mentioned in the documentation.

### Steps To Reproduce

in AppServiceProvider

```php

Relation::enforceMorphMap([
'article' => Article::class
]);

```

After call

```php

auth()->user()-> unreadNotifications

```

And get error

No morph map defined for model [App\Models\User].

Fix

```php

Relation::morphMap([
'article' => Article::class
]);

```

**Comment by driesvints** [comment]
I'm a bit confused. What do articles have to do with users?

**Comment by lee-to** [comment]
> I'm a bit confused. What do articles have to do with users?

yes, the fact of the matter is that no notifications will work afterwards, since polymorphic relations are used there and enforceMorphMap breaks them, a morphMap is needed, but nothing is said about this in the documentation


article is just an example, there can be any

**Comment by cosmastech** [comment]
It's the same with Nova.

It just reminds me to set a morph map... which I think is the intended goal.

**Comment by lee-to** [comment]
it seems to me that in the documentation it is necessary to replace the enforceMorphMap with a morphMap

**Comment by cosmastech** [comment]
I think the issue is that you need to include the map for all models. It's failing because there isn't a morph relation set up for your User model.

```php
Relation::enforceMorphMap([
    'article' => Article::class
    'user' => User::class,
]);
```

**Comment by driesvints** [comment]
I think @cosmastech is correct here. Would appreciate a PR to make this more clear in the docs.
